gobblin-utility/src/main/java/gobblin/util/concurrent/ScheduledTask.java

```diff
@@ -0,0 +1,19 @@
+/*
+ * Copyright (C) 2014-2016 LinkedIn Corp. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use
+ * this file except in compliance with the License. You may obtain a copy of the
+ * License at  http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed
+ * under the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR
+ * CONDITIONS OF ANY KIND, either express or implied.
+ */
+
+package gobblin.util.concurrent;
+
+public interface ScheduledTask<K> {
+    K getKey();
+
+    void run();
```

Comment:
I want to ensure that now exceptions can leak out other than RuntimeExceptions because depending on the implementation of the TaskScheduler, it cause break things.  For example, `ScheduledTaskExecutor` indicates: "...if any execution of the task encounters an exception, subsequent executions are suppressed..."